src/libstd/sys_common/backtrace.rs

```diff
@@ -126,7 +126,8 @@ fn filter_frames(frames: &[Frame],
     static BAD_PREFIXES_BOTTOM: &'static [&'static str] = &[
         "_ZN3std9panicking",
         "ZN3std9panicking",
-        "std::panicking",
+        "ZN3std5panic",
+        "std::panic",
```

Comment:
You need to add `std::panic` in all variations:
```
"_ZN3std5panic" // Linux
"ZN3std5panic" // Windows/GNU
"std::panic" // Windows/MSVC
```
(I'd also keep `"std::panicking"` just for clarity)